### PR TITLE
First blog post

### PR DESCRIPTION
**Pull request by milaaraujo**
Camila and Stefanni

### CODE REVIEW

File: blog/_posts/2018-07-07-brazilian-team.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- student-posts
+---
+
+### About us
+
+Hello! We are [Stefanni](https://twitter.com/steff_br) and [Camila](https://twitter.com/milasouzaaraujo), two junior software developers from Brazil based in Vancouver, Canada. We didn't know each other before the RGSoC and we are more than happy to introduce our team!
+
+#### [Stefanni](http://stefannibrasil.me/)
+
+I am a millennial that likes to how to cook (gasp) and loves coding, cats, playing sports and chocolate. I moved to Vancouver in January because my husband received a job offer to work here and we brought our two cats with us.
```
Comment (v-thakkar) [inline blog/_posts/2018-07-07-brazilian-team.md]: Hi @milaaraujo ! Thanks for the PR. See the comments below.

The sentence needs a little bit modification. '... am a millennial that likes how to cook, loves coding, cats ...'.

File: blog/_posts/2018-07-07-brazilian-team.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+I learned to code in my early years of graduation and I have a good experience with algorithms. I have good knowledge in Python, C, R... But I have almost no experience in software development, software engineering, testing, etc. So I think the Rail Girls program will be a great opportunity to be part of the Open Source community and learn a lot!
+
+### How did we meet?
+
+Because we are both relatively new to Vancouver, it was difficult to find a teammate here. So we used the RGSoC google groups to search for teammates and there was where we met each other. We went to a coffee shop and discussed about our application and we soon got along, especially because we both have cats =).
+
+Thanks, RGSoc organizers for providing these mediums for us to find a teammate :)
```
Comment (v-thakkar) [inline blog/_posts/2018-07-07-brazilian-team.md]: Change RGSoc -> RGSoC :)

File: blog/_posts/2018-07-07-brazilian-team.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+This week has been great! We are reading and collecting information from the Public Lab codebase and the API docs/tests. Also we focused on learning more about Git, RESTful API’s, and how to document code changes.
+
+We also want to thank the [Predictable Revenue](https://predictablerevenue.com/) for letting us use their space to work. Thiago, one of our coaches, works there and we asked if we could work there. The office is great and it’s located at the historical site of Vancouver, a central location in the city.
+
+![Brazilian Housewives Coding Team](/img/blog/2018/2018-07-09-brazilian-housewives-coding-team.jpg)
+<div class="image-credits"><b>Camila, Julio, Stefanni and Thiago on our first day! picture taken by Stefanni Brasil</b></div>
+
+### Meet our Amazing Team
+
+** Coaches: **
```
Comment (v-thakkar) [inline blog/_posts/2018-07-07-brazilian-team.md]: Did you intend to put these titles [coaches, mentor etc] in bold? Then you'll need to remove the space between asterisks and text. If not, then it's fine.

Also, for the consistency purpose either use all titles with colon or without colon.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by milaaraujo** [comment]
Hi, @nerdyvaishali! Thank you for reviewing our blog post. I've committed a new version.

**Review by v-thakkar — APPROVED** [review]

**Comment by v-thakkar** [comment]
Thanks @milaaraujo ! Looks good to me.

**Comment by anasofiapinho** [comment]
First student's blog post to be merged! Go Team Brazilian Housewives Coding! 🙌 cc/ @milaaraujo

**Comment by milaaraujo** [comment]
Thank you, @anasofiapinho!

**Pull request merged**